### DISCUSSION TITLE
Parameters Table

### DISCUSSION THREAD

**Discussion by pcosme** [pcosme/TETHYS-Graphene-Hydrodynamic-Simulation#67] [Theory discussion]
| Parameter   |      Description    |   Relations  | Requirements | Typical values | 
| :---:   |      :---      |  --- | :---:| ---  |
| v<sub>F</sub>   |  Fermi velocity<sup>(a)</sup>   |  |   |  1x10<sup>6</sup> ms<sup>-1</sup>  |
| E<sub>F</sub>   |  Fermi level    | = &planck;v<sub>F</sub>(&pi;n)<sup>1/2</sup>  |  k<sub>B</sub>T&Lt;E<sub>F</sub> &Lt; t &#8798;3 eV |  0.0658 n<sup>1/2</sup>  eV <sup>&dagger;</sup> |
| &mu;  |   Carrier mobility  |   |   | 2000~5000 cm<sup>2</sup>/Vs   |
| &tau;  |   Relaxation time  | = &lambda;<sub>i</sub>/v<sub>F</sub>  |   | ~5ps   (0.1 ~ 10ps) |
| 1/&tau;  |   Collision frequency<sup>(a)</sup>  |   |   | ~0.2THz  |
| &lambda;<sub>i</sub>  |  Mean free path (impurities)     | |  |  ~5&mu;m |
| &lambda;<sub>ee</sub>  | Mean free path (electrons)       ||   | ~10nm  |
| v<sub>sat</sub>  |   Saturation velocity  |     ~0.5v<sub>F</sub>| | 0.5x10<sup>6</sup> ms<sup>-1</sup>   |
| &nu;<sub>s</sub>  |   Shear kinematic viscosity<sup>(a)</sup>     ||   |  &asymp; 0.2  m<sup>2</sup>s<sup>-1</sup>  |
| &nu;<sub>o</sub>  |   Odd kinematic viscosity     || broken time symmetry (B&ne;0)  |  &lsim; 0.1  m<sup>2</sup>s<sup>-1</sup>  |
| C<sub>g</sub> | Capacitance per area   |  = &epsilon;/d<sub>0</sub>|  | 0.1~0.6 &mu;Fcm<sup>-2</sup>  |
| L |   Channel length<sup>(b)</sup>  |  | &lambda;<sub>ee</sub> < L <  &lambda;<sub>i</sub>    | |
| W |  Channel width   |  | &lambda;<sub>ee</sub> < W <  &lambda;<sub>i</sub>    | |
| x:y |  Aspect ratio<sup>(a)</sup>   | = L/W |    | |
| v<sub>0</sub> |   Mean drift velocity<sup>(b)</sup>  |  |  &Lt; v<sub>F</sub> and < v<sub>sat</sub>| ~1x10<sup>5</sup> ms<sup>-1</sup>|
| n<sub>0</sub> |   Mean carrier density<sup>(b)</sup>  |  | | ~ 1x10<sup>12</sup> cm<sup>-2</sup> |
| m<sup>&Star;</sup> | Effective mass  | = &planck;(&pi;n)<sup>1/2</sup>/v<sub>F</sub> | | ~0.02m<sub>e</sub> |
|  S |  Plasmons velocity<sup>(a)</sup>   | = ev<sub>F</sub> n<sub>0</sub><sup>1/4</sup>/&planck;C<sub>g</sub><sup>1/2</sup>&pi;<sup>1/2</sup>    |  > v<sub>F</sub> |1.17x10<sup>6</sup> n<sub>0</sub><sup>1/4</sup>C<sub>g</sub><sup>-1/2</sup>  ms<sup>-1</sup> <sup>&dagger;</sup> |
| Re  |    Reynolds number    |= v<sub>0</sub>L/&nu;<sub>s</sub>   |   | |
| t<sub>0</sub> |  Time scale<sup>(b)</sup>  | = L/v<sub>0</sub>   |  
| &omega;<sub>0</sub> |  Frequency scale<sup>(b)</sup> | = v<sub>0</sub>/L   |   |  |
| &omega;<sub>c</sub> |  Cyclotron frequency<sup>(a)</sup>  | = eB/m<sup>&Star;</sup>   |  B&Lt;1 T | &Lt; 9 THz |
| &omega;<sub>DS</sub> |   Dyakonov-Shur frequency  |  ~ &pi;S/2L  |   |  |
| &gamma;<sub>DS</sub> |  Dyakonov-Shur growth rate   | ~0.75 v<sub>0</sub>/L | > 1/&tau;  |  |
| U<sub>0</sub> |   Gate mean voltage  | = en<sub>0</sub>/C<sub>g</sub>  | | 160 n<sub>0</sub>/C<sub>g</sub> mV <sup>&dagger;</sup> |
| I<sub>0</sub> |   Typical channel current  | = en<sub>0</sub>v<sub>0</sub>W | <1.3 mA/cm for L=25 &mu;m  | 160 n<sub>0</sub>v<sub>0</sub>W &micro;A <sup>&Dagger;</sup> |
| B<sub>P</sub> | Bohm potential parameter | = (1/32&pi;)(v<sub>F</sub>/v<sub>0</sub>)<sup>2</sup>(1/L<sup>2</sup>n<sub>0</sub>) | 0.1 &mu;m < L < 0.2 &mu;m | 0.002 < B<sub>P</sub> < 0.010 |

<p><sup>(a)</sup> User defined parameter</p>
<p><sup>(b)</sup> Normalisation parameter</p>
<p> <sup>&dagger;</sup>If the density  n<sub>0</sub> is given in 10<sup>12</sup> cm<sup>-2</sup>  and the capacitance C<sub>g</sub> in &mu;Fcm<sup>-2</sup> </p>
<p> <sup>&Dagger;</sup>If the density  n<sub>0</sub> is given in 10<sup>12</sup> cm<sup>-2</sup>, the drift velocity v<sub>0</sub> in 10<sup>5</sup> ms<sup>-1</sup> and the channel width W in &micro;m </p>

**Comment by ifgrdo** [comment]
Fica super organizado assim, obrigado!

**Comment by pcosme** [comment]
For easer usage here you can find a [GoogleDocs Spreadsheet](https://docs.google.com/spreadsheets/d/16fBLPWHThss5pwo-fvUSrk9_WVuIkp9tcs7UAC0rzGk/edit?usp=sharing) that computus the derived quantities and nondimensional parameters

**Comment by pcosme** [comment]
Aproveito para chamar a atenção dos newcommers @joaorebelo-megum @RedCatiKo0  @RuiMPMartins @diogo-s-simoes para esta página :wink: 
Podem usar e claro podem e devem dar update à tabela também com as vossas constantes

**Reply by RedCatiKo0** [reply]
Já adicionei a constante que calculei no final da tabela :)
